### PR TITLE
update-contacts endpoint-bugs present

### PR DESCRIPTION
**Pull request by aksayushx**

### CODE REVIEW

**Comment by nishantkr18** [comment]
Where's the chopper generated file? Did u run `
flutter packages pub run build_runner watch`?

**Comment by aksayushx** [comment]
I did, but it ran into an error. So I used `flutter packages pub run build_runner watch --delete-conflicting-outputs` and it succeeded. I ll add the generated files soon.

**Comment by nishantkr18** [comment]
> I did, but it ran into an error. So I used `flutter packages pub run build_runner watch --delete-conflicting-outputs` and it succeeded. I ll add the generated files soon.

Yeah, U had to do that.. forgot to mention :)

**Comment by nishantkr18** [comment]
Generally these changes do not work the first time, due to some code error somewhere.. so we generally make the API call somewhere in the code to verify if it works, in the same PR.. Would u mind adding the call in the `create` page. U might want to add a text field or a drop down list for this in the create page..

File: lib/model/built_post.dart [hunk 1]
```diff
@@ -349,3 +349,14 @@ abstract class BuiltTeamMemberPost
   static Serializer<BuiltTeamMemberPost> get serializer =>
       _$builtTeamMemberPostSerializer;
 }
+
+// !--------------------------------------------------------------------------------------------------------------------
+abstract class BuiltContacts
+    implements Built<BuiltContacts, BuiltContactsBuilder> {
+  @nullable
+  BuiltList<ContactPost> get contacts;
+
```
Comment (nishantkr18) [inline lib/model/built_post.dart:358]: BTW, I guess integers are passed as body in thePost request and not `ContactPost`. You might want to confirm this by making the call :)

File: pubspec.lock [hunk 2]
```diff
@@ -21,28 +21,28 @@ packages:
       name: archive
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.0.11"
+    version: "2.0.13"
```
Comment (nishantkr18) [inline pubspec.lock:24]: Also, if possible, avoid these.. it might create dependency issues for other developers

**Comment by aksayushx** [comment]
I will do the needful soon

File: lib/data/post_api_service.dart [hunk 3]
```diff
@@ -72,6 +72,12 @@ abstract class PostApiService extends ChopperService {
   Future<Response<BuiltList<BuiltWorkshopSummaryPost>>> searchWorkshopByDate(
       @Body() BuiltWorkshopSearchByDatePost body);
 
+  @Patch(path: '/workshops/{id}/update-contacts/')
```
Comment (mohit-mangal) [inline lib/data/post_api_service.dart:75]: There is no ***patch*** request for updating contacts. Only a ***put*** request

Comment (mohit-mangal) [reply lib/data/post_api_service.dart:75]: And please update your repo as there have been many changes since this pr

Comment (aksayushx) [reply lib/data/post_api_service.dart:75]: Sure

**Pull request closed without merging**